swap-create@.service

```diff
@@ -0,0 +1,14 @@
+# SPDX-License-Identifier: MIT
+
+[Unit]
+Description=Create swap on /dev/%i
+Wants=systemd-modules-load.service
+After=systemd-modules-load.service
+After=dev-%i.device
+DefaultDependencies=false
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStartPre=-modprobe zram
```

Comment:
I'm not sure if this is necessary, seeing as we schedule ourselves after not only `systemd-modules-load.service`, but `dev-%i.device` as well, which can only be provided by the zram module.

Reply:
Hmm, indeed. Let's keep this for now, it doesn't hurt.

Reply:
Why not reference modprobe@zram.service instead?

Reply:
Because leaving it as-is is cleaner than changing it one way now, and the other in the next patch I'm gonna send (cf. #14).